model/event/subscriptions.go

```diff
@@ -238,14 +248,29 @@ func (s *Subscription) String() string {
 	return out
 }
 
-func FindSubscriptionsByOwner(owner string) ([]Subscription, error) {
+func FindSubscriptionsByOwner(owner string, ownerType OwnerType) ([]Subscription, error) {
 	if len(owner) == 0 {
 		return nil, nil
 	}
+	if !IsValidOwnerType(string(ownerType)) {
+		return nil, errors.Errorf("%s is not a valid owner type", ownerType)
```

Comment:
BadRequest

Reply:
This is the model package. The route already returns 400 if it fails to validate